### ISSUE TITLE
Linter: deprecate the use of py27, py36, encorage py==27, py==36

### ISSUE THREAD

**Issue by croth1** [conda-forge/conda-forge-webservices#254]
Using the selectors py27 and py36 intuitively suggests that py37 and py38 are available too.

However the use of those selectors is deprecated and py37 and py38 selectors do not exist. Trying to use the selector py37 leads to unexpected behavior: The selector is simply ignores.

I think it may make sense to enforce using the new `py==<version>` syntax during linting.

**Comment by scopatz** [comment]
Thanks! This is probably best opened on conda-smithy, where the code for the linter lives

**Comment by croth1** [comment]
@scopatz, fair enough! Do you have enough power to [transfer](https://help.github.com/en/articles/transferring-an-issue-to-another-repository) the issue to the other repo or should I close it and open a new one?
